### PR TITLE
Bug 1629 recentfile

### LINKED ISSUE #1629: Recents - remove file from recents list if selected for load and not found
See title. When .side files are removed, they remain in the recents list. I considered scanning the recents and only showing if exists, but that can be expensive with network shares in play, so better to wait until the user selects the file, and remove individually if not found.

### PR DESCRIPTION
**Pull request by ghost**
**Thanks for contributing to the Selenium IDE!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://github.com/SeleniumHQ/selenium-ide/blob/master/CONTRIBUTING.md) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

closes #1629 

### Description
A warning pop is added to show the user recent file is missing and the file will be removed from recent list 

### Motivation and Context
To remove the recent file in list if it is not present 

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [x ] I have read the [contributing](https://github.com/SeleniumHQ/selenium-ide/blob/master/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x ] All new and existing tests passed.

### CODE REVIEW

**Comment by toddtarsi** [comment]
Hey! I suspected it might be you. Want a review?

File: packages/selenium-ide/src/main/session/controllers/Projects/index.ts [hunk 1]
```diff
@@ -116,7 +116,11 @@ export default class ProjectsController {
     return starterProject
   }
 
-  async load(filepath: string): Promise<CoreSessionData | null> {
+  async load(filepath: string): Promise<CoreSessionData | null > {
+    const fileExists = await fs.stat(filepath)
+    .then(() => true)
+    .catch(() => false);
```
Comment (toddtarsi) [inline packages/selenium-ide/src/main/session/controllers/Projects/index.ts:122]: Nice call on catching the error like this, v smart!

**Comment by toddtarsi** [comment]
@mrgamedev07 - Let me know when you'd like a review! I'd be happy to approve this as is, but I just want to get the a-okay from you. 

Also, you probably want to do the following to autogenerate a release of the selenium-ide binary:

1. Increment the version in the packages you want to release a new version of (packages/selenium-ide)
2. Increment any dependent packages (none here, but this is kinda obnoxious with side-runtime)
3. Run `pnpm i -r` to regen the lockfile and commit as well (ci release action dumps on install step if this gets missed)


Nice work as always! Very smart with that catch. Thank you

**Comment by mrgamedev07** [comment]
Yes, I definitely like a review, Yup all your points are noted.

**Review by toddtarsi — APPROVED** [review]
LGTM 👍

**Comment by toddtarsi** [comment]
@mrgamedev07 - You're approved. I see the side runner tests complaining. Ugh, it's insane how we talk about test flakiness and how its easy to tame and then we base a whole suite off of the UX at google.com never changing 🙄

**Comment by toddtarsi** [comment]
@mrgamedev07 - Would you like me to merge or are you able to? Further, want me to increment so a release rolls out with your fixes?

**Comment by mrgamedev07** [comment]
I am not able to @toddtarsi , u can merge it .

**Pull request merged**